frontend/src/pages/team/Devices/dialogs/ProvisioningCredentialsDialog.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-    <ff-dialog ref="dialog" header="Device Provisioning Credentials">
+    <ff-dialog ref="dialog" header="Device Provisioning Configuration">
         <template v-slot:default>
             <form class="space-y-6 mt-2">
                 <p class="text-sm text-gray-500">
                     To auto provision your devices on the platform, use the following
-                    credentials. Make a note of them as this is the only
-                    time you will see them.
+                    configuration. Make a note of them as this is the only
+                    time you will see it.
```

Comment:
```suggestion
                    configuration. Make a note of it as this is the only
                    time you will see it.
```

Sorry Joe, last one. - I missed a plural/singular mismatch.

Feel free to 1up it.